src/content/structured/get-started/gatsby.mdx

```diff
@@ -0,0 +1,77 @@
+---
+path: "/get-started/install-components/gatsby"
+
+navPriority: 7
+
+date: "2023-01-19"
+
+title: "Gatsby"
+
+subTitle: "How to use the components in a Gatsby project."
+
+contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/get-started/gatsby.mdx"
+---
+
+## Step one
+
+In the root of your project:
```

Comment:
What is this suggesting? 'In the root write ...' or 'In the root find ...'. Need to add a verb to add context to the instruction.